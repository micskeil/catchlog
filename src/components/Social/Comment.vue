<template>
  <div>
    <form class="container row">
      <label class="font-weight-bold col-12 d-none" for="comment"
        >Komment:</label
      >
      <div class="d-flex">
        <input
          class="pr-3"
          type="text"
          id="comment"
          name="comment"
          placeholder="Hozzászólás írása..."
        />
        <img class="ml-3 " src="../../assets/send.png" width="24" height="24" />
      </div>
    </form>
  </div>
</template>

<script>
export default {};
</script>

<style lang="scss" scoped>
$color: #2c3e50;
div {
  width: 100%;

  form {
    width: 100%;
    div {
      input {
        border: none;
        width: 100%;
        border-bottom: 1px solid $color;
        border-radius: 0px;
      }
      img {
        opacity: 0.9;
      }
      select {
        border: none;
        border-bottom: 1px solid $color;
        border-radius: 0;
      }
    }
    .warning {
      color: white;
      background-color: #c79136;
    }
  }
}
</style>
