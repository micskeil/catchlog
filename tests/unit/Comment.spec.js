import { shallowMount } from "@vue/test-utils";
import Comment from "../../src/components/Social/Comment";

describe("Comment", () => {
  it("should give back error with no postId ", () => {
    const propsData = { postId: undefined };
    const wrapper = shallowMount(Comment, { propsData });
    expect(wrapper.element).toMatchSnapshot();
  });

  it("should give back error with no postId ", () => {
    const propsData = { uid: "" };
    const wrapper = shallowMount(Comment, { propsData });
    expect(wrapper.element).toMatchSnapshot();
  });
});
