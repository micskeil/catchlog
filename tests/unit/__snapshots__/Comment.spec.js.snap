// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment should give back error with no postId  1`] = `
<div>
  <form
    class="container row"
  >
    <label
      class="font-weight-bold col-12 d-none"
      for="comment"
    >
      Komment:
    </label>
    <div
      class="d-flex"
    >
      <input
        class="pr-3"
        id="comment"
        name="comment"
        placeholder="Hozzászólás írása..."
        type="text"
      />
      <img
        class="ml-3 "
        height="24"
        width="24"
      />
    </div>
  </form>
</div>
`;

exports[`Comment should give back error with no postId  2`] = `
<div
  uid=""
>
  <form
    class="container row"
  >
    <label
      class="font-weight-bold col-12 d-none"
      for="comment"
    >
      Komment:
    </label>
    <div
      class="d-flex"
    >
      <input
        class="pr-3"
        id="comment"
        name="comment"
        placeholder="Hozzászólás írása..."
        type="text"
      />
      <img
        class="ml-3 "
        height="24"
        width="24"
      />
    </div>
  </form>
</div>
`;
